scss/profileLessons.scss

```diff
@@ -0,0 +1,16 @@
+.lessonImage_container{
```

Comment:
not sure how i feel about the camelcase class names? lol i know its a little nitpick, but maybe we should match the current class name styles?

Reply:
\

> not sure how i feel about the camelcase class names? lol i know its a little nitpick, but maybe we should match the current class name styles?

I am following current classNames structure, i followed LessonCard template in name convention for classes

Reply:
> not sure how i feel about the camelcase class names? lol i know its a little nitpick, but maybe we should match the current class name styles?

https://share.getcloudapp.com/GGuk2mXN

Reply:
we don't have any camelCase class name though? it should be more like `lesson-image__container`

Reply:
hmmm got it